app/scenes/Settings/Features.tsx

```diff
@@ -57,6 +58,22 @@ function Features() {
           />
         </SettingRow>
       )}
+      {team.avatarUrl && !isCloudHosted && (
+        <SettingRow
+          name="publicBranding"
+          label={t("Public branding")}
+          description={t(
+            "Whether to use team logo across entire application for branding, take Login screen as one example."
```

Comment:
```suggestion
            "Show your team’s logo on public pages like login and shared documents."
```

Can be a follow up PR but I want to use this to also show the logo at the top of the left sidebar on shared documents such as: https://wiki.generaloutline.com/s/770a97da-13e5-401e-9f8a-37949c19f97e